src/integrationTest/resources/zipcontents/supplementary_evidence_with_ocr/metadata.json

```diff
@@ -22,5 +22,8 @@
     }
   ],
   "payments": [
+    {
+      "document_control_number": "1111002"
+    }
```

Comment:
there should be a scenario with empty payments ;). is there one?

Reply:
Added the test scenario